apps/i18n/common/en_us.json

```diff
@@ -207,6 +207,7 @@
   "coursesHeadingTeacher": "Teach with Code Studio",
   "coursesUiTipsTeacherCourses": "We've updated this page just for teachers. Find the next course for your classroom here.",
   "createAccount": "Create an account",
+  "createAccountApplabDesc": "Want to share your project? Or, work on it later? Join Code.org! With an account you can also publish projects to your phone or to the public gallery.",
```

Comment:
Let's change to this. They key part of this desc is that creating an account saves progress.

`With an account, you can save your progress. You can work on your App Lab project later, play it on your phone, or share it with friends.`